runtests.py

```diff
@@ -1,24 +1,45 @@
 #!/usr/bin/env python
 import sys
 import os
+import contextlib
 
-try:
-    from django import setup
-except ImportError:
-    import traceback
-    traceback.print_exc()
-    msg = "To fix this error, run: pip install -r requirements_test.txt"
-    raise ImportError(msg)
 
-from django.test.utils import get_runner
-from django.conf import settings
-os.environ["DJANGO_SETTINGS_MODULE"] = "tests.readonly_project.settings"
-setup()
+@contextlib.contextmanager
+def cover():
+    do_coverage = "COVERAGE" in os.environ
+
+    if do_coverage:
+        import coverage
```

Comment:
That's perfectly valid for flake8. Note that it isn't if you are not in a block.